lib/kumascript/firelogger.js

```diff
@@ -0,0 +1,83 @@
+/*jshint node: true*/
```

Comment:
FYI. This is a copy of https://github.com/lmorchard/node-firelogger/blob/master/firelogger.js but updated to use the latest methods of the Request and Response objects of `express` (specifically, the `get` and `set` methods). Also a comment has been added to the top of the file to suppress "implied global module" errors when running `jshint`.